src/kv/LevelDBStore.cc

```diff
@@ -218,6 +218,22 @@ int LevelDBStore::get(
   return 0;
 }
 
+int LevelDBStore::get(const string &prefix, 
+		  const string &key,
+		  bufferlist *value)
+{
+  utime_t start = ceph_clock_now(g_ceph_context);
+  KeyValueDB::Iterator it = get_iterator(prefix);
+  it->lower_bound(key);
+  if (it->valid() && it->key() == key) {
+      *value = it->value();
```

Comment:
i think we need to else return -ENOENT.  can probably mostly cut&paste from here:

https://github.com/liewegas/ceph/blob/wip-kv/src/kv/RocksDBStore.cc#L296